pkg/lib/operation/project/remote/table/import/operation.go

```diff
@@ -43,36 +46,42 @@ func Run(ctx context.Context, o Options, d dependencies) (err error) {
 			return err
 		}
 
-		_, err = d.KeboolaProjectAPI().CreateTableFromFileRequest(o.TableKey, o.FileKey, getCreateOptions(&o)...).Send(ctx)
-		if err != nil {
-			rb.Invoke(ctx)
-			return err
+		if (o.Columns != nil && !o.WithoutHeaders) || (o.WithoutHeaders && o.Columns == nil) {
```

Comment:
Why did we connect `columns` and `headers` flags? I can have csv file without headers and specified `columns` flag. Now it says that I cannot use because of missing `without headers` flag. Do we want to return error or just ignore the `without headers` as before?
I would say that only `2nd` condition is correct. First one is too restrictive, the csv can or does not need to have header.

Reply:
[7916294](https://github.com/keboola/keboola-as-code/pull/1841/commits/7916294c1dee38c2b2739525ce8e4098de12a529)